project_access_tokens.go

```diff
@@ -34,16 +34,16 @@ type ProjectAccessTokensService struct {
 //
 // GitLab API docs: https://docs.gitlab.com/ee/api/resource_access_tokens.html
 type ProjectAccessToken struct {
-	ID          int              `json:"id"`
-	UserID      int              `json:"user_id"`
-	Name        string           `json:"name"`
-	Scopes      []string         `json:"scopes"`
-	CreatedAt   *time.Time       `json:"created_at"`
-	ExpiresAt   *ISOTime         `json:"expires_at"`
-	Active      bool             `json:"active"`
-	Revoked     bool             `json:"revoked"`
-	Token       string           `json:"token"`
-	AccessLevel AccessLevelValue `json:"access_level"`
+	ID          int               `json:"id"`
+	UserID      int               `json:"user_id"`
+	Name        string            `json:"name"`
+	Scopes      []string          `json:"scopes"`
+	CreatedAt   *time.Time        `json:"created_at"`
+	ExpiresAt   *ISOTime          `json:"expires_at"`
+	Active      bool              `json:"active"`
+	Revoked     bool              `json:"revoked"`
+	Token       string            `json:"token"`
+	AccessLevel *AccessLevelValue `json:"access_level"`
```

Comment:
I'm sorry, but only the one in the options struct should be a pointer. This one should just be a value instead:
```suggestion
	AccessLevel AccessLevelValue `json:"access_level"`
```